edge-hub/src/Microsoft.Azure.Devices.Edge.Hub.Service/Hosting.cs

```diff
@@ -59,10 +99,29 @@ public class Hosting
                         serviceCollection.AddSingleton(configuration);
                         serviceCollection.AddSingleton(dependencyManager);
                     })
-                .UseStartup<Startup>();
+                .UseStartup<Startup>()
+                .UseSetting(WebHostDefaults.ApplicationKey, typeof(TwinsController).Assembly.GetName().Name);
```

Comment:
would you explain a bit more about UseSetting? and share any reference if have?

Reply:
We need it in order to find the TwinsController in the Http project. Dotnet/Aspnetcore 3 has issues finding controllers in separate projects. There is a lot of context in this issue:
https://github.com/dotnet/aspnetcore/issues/14952